config/enabled.yml

```diff
@@ -420,6 +420,7 @@ Style/AsciiComments:
   Description: 'Use only ascii symbols in comments.'
   StyleGuide: '#english-comments'
   Enabled: true
+  AllowedChars: []
```

Comment:
Default configuration should go in the `default.yml` file. 🙂

Reply:
Thank for review. Fixed.